Makefile

```diff
@@ -32,16 +32,16 @@ build-softhsm-container:
 test-container:
 	docker run --name autograph-dev --rm -u 0 --net host app:build make -C /go/src/go.mozilla.org/autograph test
 
-
 run-container:
 	docker run --name autograph-dev --rm -d --net host app:build
 
+compose:
+	docker-compose up --build
+
 vendor:
 	govend -u --prune
 	#go get -u github.com/golang/dep/...
 	#dep ensure -update
-	# https://github.com/ThalesIgnite/crypto11/issues/9
-	git checkout -f 2210ea80470825094edf8235b35f9565c7940555 vendor/github.com/ThalesIgnite/crypto11/
```

Comment:
when we deploy we should make sure we don't regress on the bug that pinning this fixed

Reply:
Is this asking for change on this PR or just a personal reminder to test this?

Reply:
Personal reminder, but please remind @milescrabill and me for the next deploy too.